### PR TITLE
Numark NS6II docs

### PR DESCRIPTION
**Pull request by Swiftb0y**
This is the mapping documentation for https://github.com/mixxxdj/mixxx/pull/11075. Please review as I don't know how much longer I can work on the mapping because my controller is starting to fall apart.

### CODE REVIEW

**Comment by Swiftb0y** [comment]
CC @acolombier since you were so kind to review the mapping.

**Comment by Swiftb0y** [comment]
Build fails because I'm depending on #685

File: source/hardware/controllers/numark_ns6ii.rst [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+13        :hwlabel:`🎧` (pfl/cue) button                                  Toggle :term:`PFL`. Notice: due to hardware constraints, this always works in numark mode: clicking a single button solos the corresponding channel, to listen to multiple channels, press and then release multiple buttons at once.
+14        :hwlabel:`A OFF B` switch                                       The :term:`crossfader` orientation switches determine to which side of the crossfader the deck is assigned. It can be set to the left or right side, or to the center where it is unaffected by the crossfader knob.
+47        :hwlabel:`FILTER`                                               This knob lets you control the Meta knob of the deck’s Quick Effect.
+38        :hwlabel:`LOAD`                                                 Load the currently selected track from the library into the deck corresponding to the channel
+48        :hwlabel:`A` / :hwlabel:`B`                                     Route Channel to left/right :ref:`Effect Unit <interface-effects>`.
+========  ==============================================================  ==========================================
```
Comment (ronso0) [inline source/hardware/controllers/numark_ns6ii.rst:169]: The odd numbering from the SVG is confusing already :zany_face:  (I presume that's from the off. manual?)
so how about we simply adopt it so the order in SVG and table match?

Comment (ronso0) [reply source/hardware/controllers/numark_ns6ii.rst:169]: Btw 48 --> 46

**Review by ronso0 — COMMENTED** [review]
I just skimmed it to see if it was easy to read. All good except the odd numbering n some places :+1:

File: source/hardware/controllers/numark_ns6ii.rst (hunk 1 above)
Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:169]: yeah, its from the original manual and I agree its suboptimal. I don't understand your proposal however. Can you elaborate?

Comment (ronso0) [reply source/hardware/controllers/numark_ns6ii.rst:169]: Let's reorder the tabe rows so the order (top -> bottom) matches the order of the numbers in the SVG.

And 48 is probably referring to 46 in the SVG.

Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:169]: So just order ascending by `No.`?

Comment (ronso0) [reply source/hardware/controllers/numark_ns6ii.rst:169]: Nope, simply same order as in th SVG

|| SVG | table |
|-|-|-|
|<td rowspan="11">image</td> | 38
|| 46
|| 7
|| 9
|| 8
|| 10
|| 11
|| 47
|| 13
|| 12

Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:169]: Ah I see. Sure.

Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:169]: fixed. Thank you.

File: source/hardware/controllers/numark_ns6ii.rst [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+21        :hwlabel:`SYNC` button                                                  Short press to match tempo and phase of other deck once. Long press to enable :ref:`Sync Lock <sync-lock>`.
+21        :hwlabel:`SHIFT` + :hwlabel:`SYNC` button                               Press to toggle :term:`quantization`.
+22        :hwlabel:`BLEEP` Button                                                 Press to play in reverse, return to where the track would be with no button press on button release.
+22        :hwlabel:`SHIFT` + :hwlabel:`BLEEP` Button (:hwlabel:`KEYLOCK`)         Toggle :term:`Keylock <key lock>`
+23        :hwlabel:`SCRATCH` button                                               Toggle platter scratching. If active, touching the jogwheel results in scratch behavior, if disabled jogwheel behaves as if only outer edge was touched.
+23        :hwlabel:`SHIFT` + :hwlabel:`SCRATCH` button                            Switch display between showing remaining/elapsed time
+24        :hwlabel:`STRIP SEARCH`                                                 Seek to absolute position in *stopped* track (does nothing for playing tracks to avoid accidental)
```
Comment (ronso0) [inline source/hardware/controllers/numark_ns6ii.rst:108]: just "accidents"?

File: source/hardware/controllers/numark_ns6ii.rst [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+    mixxx and the built-in mixer by toggling the split cue switch once.
+
+Settings
+--------
+
+================================================  ===================================================
+Label                                             Description
+================================================  ===================================================
+Use Button Backlight                              Checked: deactivated button has a slight (usually red) backlight, Unchecked: LED of deactivated button is completely off.
+Move this many tracks at once with :hwbtn:SHIFT   Amount of tracks moved in a single step when turning the :hwlabel:`BROWSE` encoder while pressing :hwlabel:`SHIFT`.
```
Comment (ronso0) [inline source/hardware/controllers/numark_ns6ii.rst:540]:
```suggestion
Move this many tracks at once with :hwlabel:`SHIFT`   Amount of tracks moved in a single step when turning the :hwlabel:`BROWSE` encoder while pressing :hwlabel:`SHIFT`.
```
you probably also need to adjust the tabel alignment with this change

File: source/hardware/controllers/numark_ns6ii.rst [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+8         Channel VuMeter                                                 loudness of signal of that specific channel (after EQ, before Filter and Faders). A white LED indicates a clipping signal (see :term:`level meter`).
+10        :hwlabel:`MID`                                                  Channel Mid EQ
+11        :hwlabel:`LOW`                                                  Channel Low/Bass EQ
+47        :hwlabel:`FILTER`                                               This knob lets you control the Meta knob of the deck’s Quick Effect.
+14        :hwlabel:`A OFF B` switch                                       The :term:`crossfader` orientation switches determine to which side of the crossfader the deck is assigned. It can be set to the left or right side, or to the center where it is unaffected by the crossfader knob.
+13        :hwlabel:`🎧` (pfl/cue) button                                  Toggle :term:`PFL`. Notice: due to hardware constraints, this always works in numark mode: clicking a single button solos the corresponding channel, to listen to multiple channels, press and then release multiple buttons at once.
```
Comment (ronso0) [inline source/hardware/controllers/numark_ns6ii.rst:167]: Umm, nice character but hard to see with the dark button bg.
image

Can't propose an alternative so it's okay I guess

**Review by ronso0 — CHANGES_REQUESTED** [review]
LGTM otherwise!

File: source/hardware/controllers/numark_ns6ii.rst [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+The Numark NS6II is an all-in-one USB :term:`MIDI` controller with an integrated audio interface.
+It has controls for 2 decks that can be toggled between decks to play with 4 decks.
+Its jogwheels feature a built-in screen highlighting basic information on the currently
+playing track of each deck.
+
+-  `Manufacturer’s product page <http://www.numark.com/product/ns6ii>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-ns6ii>`__
+-  `User Guide <https://www.numark.com/images/product_downloads/NS6II-UserGuide-v1.1.pdf>`__
+
+.. versionadded:: 2.5.0
```
Comment (ronso0) [inline source/hardware/controllers/numark_ns6ii.rst:29]:
```suggestion
.. versionadded:: 2.5.1
```

File: source/hardware/controllers/numark_ns6ii.rst (hunk 4 above)
Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:167]: yeah, I agree its suboptimal, but the exact icon will depend on the user anyways.

File: source/hardware/controllers/numark_ns6ii.rst (hunk 5 above)
Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:29]: right, that ship has sailed :sweat_smile:

File: source/hardware/controllers/numark_ns6ii.rst (hunk 2 above)
Comment (Swiftb0y) [reply source/hardware/controllers/numark_ns6ii.rst:108]: yeah, didn't know how to finish the sentence and then just forgot. "accidents" is good.

**Comment by Swiftb0y** [comment]
done

**Review by ronso0 — APPROVED** [review]
LGTM, thank you!

**Comment by ronso0** [comment]
waiting for https://github.com/mixxxdj/mixxx/pull/11075

**Comment by Swiftb0y** [comment]
Thank you.

**Comment by ronso0** [comment]
Ah, there's no 'fixup-prevent-merge' hook in the manual repo.
Can you please squash the commits? Then this is ready to roll once the mapping is ready.

**Comment by Swiftb0y** [comment]
Yup. Thank you.

**Pull request merged**